### DISCUSSION TITLE
Temperature sensor keeps changing values

### DISCUSSION THREAD

**Discussion by dgarcia90** [Rem0o/FanControl.Releases#1643] [General]
Hello, I've been using this program for a while now. I've been able to configure all the fan controls and everything works great.

One thing I'm trying to do is configure a sensor to show me GPU hotspot and CPU temps and I want to see the MAX temps of those sensors.

However when I do that the temps keep changing. Instead of changing when the MAX temps increases, it changes when gpu hostpot and cpu temp change, no matter if they go higher or lower.

Here are a couple of screenshots of from the gpu hotspot sensor

<img width="284" alt="image" src="https://user-images.githubusercontent.com/122970941/227154461-cf04561f-58f6-40ee-bd19-f4b4d4791c42.png">

<img width="294" alt="image" src="https://user-images.githubusercontent.com/122970941/227154328-ef8e4356-0384-4d99-931d-5d1faa5a602e.png">

I don't know if this is some sort of bug or this is the intended behaviour

Thanks in advance!

**Comment by Rem0o** [comment]
It's a mix sensor, it is meant to add multiple sensors onto it, and it will output the maximum of the selected sensor(s) (plural).  It's not a monitoring tool to keep track of the maximum temp.

**Reply by dgarcia90** [reply]
Thanks for the answer! :)
